packages/block-editor/src/components/button-block-appender/style.scss

```diff
@@ -3,9 +3,10 @@
 	flex-direction: column;
 	align-items: center;
 	justify-content: center;
-	padding: $block-padding*1.5;
+	padding: $block-padding;
 	outline: $border-width dashed $dark-gray-150;
 	width: 100%;
+	height: 100%;
```

Comment:
I'm not sure about this rule as it's dependent on the parent element and this is a generic component,  should this be `auto` instead  of`100%`?

Reply:
Yes, I agree! :) 

I actually realised this after I opened the PR, but forgot to commit the update.